src/utils/hooks/useUpdateReviewResponse.ts

```diff
@@ -46,7 +46,7 @@ const computeVisibility = (reviewResponse: ReviewResponse | undefined) => {
   // consolidator
   if (
     reviewResponse.decision === ReviewResponseDecision.Agree &&
-    reviewResponse.reviewResponseLink?.decision === ReviewResponseDecision.Decline
+    reviewResponse.originalReviewResponse?.decision === ReviewResponseDecision.Decline
```

Comment:
this looks like a correct change (however would only affect lvl 3 consolidation by the looks of it), ok to have it in i think

Reply:
Hmm, I don't think this was an intended change, it doesn't really have anything to do with the issue. The commit message I wrote was "Revert unrelated changes", and now I'm not sure what it's referring to. I think I should get @nmadruga to check was this is supposed to be before I merge it.